dataframe-geo/src/main/kotlin/org/jetbrains/kotlinx/dataframe/geo/io/read.kt

```diff
@@ -0,0 +1,43 @@
+package org.jetbrains.kotlinx.dataframe.geo.io
+
+import org.geotools.data.shapefile.ShapefileDataStoreFactory
+import org.geotools.data.simple.SimpleFeatureCollection
+import org.geotools.geojson.feature.FeatureJSON
+import org.jetbrains.kotlinx.dataframe.DataFrame
+import org.jetbrains.kotlinx.dataframe.geo.GeoDataFrame
+import org.jetbrains.kotlinx.dataframe.geo.geotools.toGeoDataFrame
+import org.jetbrains.kotlinx.dataframe.io.asURL
+import java.net.URL
+
+
+fun GeoDataFrame.Companion.readGeoJson(path: String): GeoDataFrame<*> {
+    return readGeoJson(asURL(path))
```

Comment:
One thing the linter will complain about is that these should be expression functions. One `ktLintFormat` can replace them all at once tho